### PR TITLE
Java Hello World Module - closes #365

### LINKED ISSUE #365: Java Hello World Module
For Johnny Newb

Prerequisite:
- java/introduction

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Git is a distributed version-control system for tracking changes in source code
 
 ## Iptables [&xrarr;](./iptables)
 Iptables is a command line utility for configuring Linux kernel firewall.
+## Java [&xrarr;](./java)
+Java is a general-purpose programming language that is class-based, object-oriented, and designed to have as few implementation dependencies as possible.
```
Comment (jordan-grindrod) [inline topics/README.md]:
```suggestion
Java is a general-purpose programming language that is class-based, object-oriented, and designed to have as few implementation dependencies as possible. Wat a bloody weapon
```

File: topics/java/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Java
+<!--TOC_START-->
+### Contents
+- [Overview](#overview)
+- [Modules](#modules)
+
+<!--TOC_END-->
+## Overview
```
Comment (jordan-grindrod) [inline topics/java/README.md]: weapons everywhere

Comment (bob-crutchley) [reply topics/java/README.md]: fixed it

**Comment by bob-crutchley** [comment]
This was for demo purposes - closing now

**Pull request closed without merging**